client/signup/steps/survey/index.jsx

```diff
@@ -59,12 +60,13 @@ export default React.createClass( {
 		const stepOneClickHandler = ( event ) => {
 			event.preventDefault();
 			event.stopPropagation();
+			this.setState( { surveyFirstQuestion: vertical.value } );
```

Comment:
@sirbrillig Yes! I already figured out this was way more complex than it needed to be, just hadn't pushed yet.